### ISSUE TITLE
[bug]: The input type:file display does not match the documentation

### ISSUE THREAD

**Issue by FelixR11** [shadcn-ui/ui#4554]
### Describe the bug

The input type:file display does not match the documentation. There seems to be a problem with the className for type: file.

### Affected component/components

Input

### How to reproduce

1. install shadcn [installation](https://ui.shadcn.com/docs/installation/vite)
2. npx shadcn-ui@latest add input
3. create input_file.tsx file
4. npm run dev
5. open localhost

[Documentation:](https://ui.shadcn.com/docs/components/input#file)
![Screenshot 2024-08-11 215408](https://github.com/user-attachments/assets/251c3b30-d08d-4a9e-8592-339e3e999348)

After i try:
![Screenshot 2024-08-11 214559](https://github.com/user-attachments/assets/0b022a89-952b-4c0a-8226-4776669978b9)

code:
- input.tsx
![Screenshot 2024-08-11 220101](https://github.com/user-attachments/assets/d9e9c585-ea78-4f9e-b47d-6ef1064f144a)

- input_file.tsx
![Screenshot 2024-08-11 220340](https://github.com/user-attachments/assets/be76b23a-670c-4aa9-8aaa-14845e44a221)

### System Info

```bash
OS: windows 10
Browser: Chrome
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by imMatheus** [comment]
I feel like this is most likely not really an issue with ShadcnUI but maybe more with some other configs you have. How does your `index.css` look like? I fear you might have some other styling there that is overriding here.

**Comment by FelixR11** [comment]
> I feel like this is most likely not really an issue with ShadcnUI but maybe more with some other configs you have. How does your `index.css` look like? I fear you might have some other styling there that is overriding here.

I don't think I added anything that caused this problem. But here is my index.css file

index.css:
/* Tailwind Start */
@tailwind base;
@tailwind components;
@tailwind utilities;
/* Tailwind End */

/* Shadcn Start */
@layer base {
  :root {
    --background: 0 0% 100%;
    --foreground: 222.2 84% 4.9%;

    --card: 0 0% 100%;
    --card-foreground: 222.2 84% 4.9%;

    --popover: 0 0% 100%;
    --popover-foreground: 222.2 84% 4.9%;

    --primary: 222.2 47.4% 11.2%;
    --primary-foreground: 210 40% 98%;

    --secondary: 210 40% 96.1%;
    --secondary-foreground: 222.2 47.4% 11.2%;

    --muted: 210 40% 96.1%;
    --muted-foreground: 215.4 16.3% 46.9%;

    --accent: 210 40% 96.1%;
    --accent-foreground: 222.2 47.4% 11.2%;

    --destructive: 0 84.2% 60.2%;
    --destructive-foreground: 210 40% 98%;

    --border: 214.3 31.8% 91.4%;
    --input: 214.3 31.8% 91.4%;
    --ring: 222.2 84% 4.9%;

    --radius: 0.5rem;
  }

  .dark {
    --background: 222.2 84% 4.9%;
    --foreground: 210 40% 98%;

    --card: 222.2 84% 4.9%;
    --card-foreground: 210 40% 98%;

    --popover: 222.2 84% 4.9%;
    --popover-foreground: 210 40% 98%;

    --primary: 210 40% 98%;
    --primary-foreground: 222.2 47.4% 11.2%;

    --secondary: 217.2 32.6% 17.5%;
    --secondary-foreground: 210 40% 98%;

    --muted: 217.2 32.6% 17.5%;
    --muted-foreground: 215 20.2% 65.1%;

    --accent: 217.2 32.6% 17.5%;
    --accent-foreground: 210 40% 98%;

    --destructive: 0 62.8% 30.6%;
    --destructive-foreground: 210 40% 98%;

    --border: 217.2 32.6% 17.5%;
    --input: 217.2 32.6% 17.5%;
    --ring: 212.7 26.8% 83.9%;
  }
}

@layer base {
  * {
    @apply border-border;
  }
  body {
    @apply bg-background text-foreground;
  }
}
/* Shadcn End */

/* Custom Start */
/* font style and change theme transistion Start */
body * {
  @apply font-mono;
  @apply transition-colors duration-75 ease-in-out;
}
/* font style and change theme transistion End */

/* Remove input number arrow Start */
.remove-arrow::-webkit-inner-spin-button,
.remove-arrow::-webkit-outer-spin-button {
  -webkit-appearance: none;
  margin: 0;
}

.remove-arrow {
  -moz-appearance: textfield;
}
/* Remove input number arrow End */
/* Custom End */

**Comment by Jay-Karia** [comment]
@FelixR11 
Tried reinstalling the component ?

**Comment by FelixR11** [comment]
I already re-add the component and overwrite it. The result is still the same as what I gave above.

terminal (pwsh):
```
npx shadcn-ui@latest add input
√ Component input already exists. Would you like to overwrite? ... yes
✔ Done.
```

**Comment by Jay-Karia** [comment]
Try installing manually

**Comment by FelixR11** [comment]
I tried to reinstall with new files, here are the results.

result (dark theme):
![Screenshot 2024-08-18 012624](https://github.com/user-attachments/assets/bcadba71-2faf-414f-b126-34bd9f8498a6)

after I checked again the files that might have css styles, I forgot I added the css style script from flowbite in index.html.

I use flowbite to create a sidebar that has a navbar. The sidebar has a hidden feature when the display has a smartphone screen size and it uses javascript in it so I added the script. [flowbite sidebar](https://flowbite.com/docs/components/sidebar/#sidebar-with-navbar)

flowbite script:
```
<link href="https://cdnjs.cloudflare.com/ajax/libs/flowbite/2.3.0/flowbite.min.css" rel="stylesheet" />
<script src="https://cdnjs.cloudflare.com/ajax/libs/flowbite/2.3.0/flowbite.min.js"></script>
```

I apologize for my mistake and thank you for your help.
